script/input/1/mainnet.json

```diff
@@ -0,0 +1,5 @@
+{
+    "cctpTokenMessenger": "0xBd3fa81B58Ba92a82136038B25aDec7066af3155",
+    "safe": "0x22fB6fe2B9aA289D26724eCBD5a679751A4508b5",
```

Comment:
Staging safe on mainnet is `0x611C7c37F296240c2fF5a92f0B4a398B01B237c4`

Reply:
Updated